e2e/wdio/platform/tests/standard-list-item.e2e-spec.ts

```diff
@@ -0,0 +1,189 @@
+                expect(webDriver.getAttributeByName(standardListPg.sSecTypeListItem, StandardLinkData.secondaryAttr, i))
+                    .not.toBe(null, '');
+                expect(webDriver.getAttributeByName(standardListPg.sSecTypeListItem, StandardLinkData.secondaryAttr, i))
+                    .toBe(StandardLinkData.secondaryTypes[i]);
+            }
+        });
+    });
+
+    describe('Standard List Item (ByLine)- Multi Selection examples:', function() {
+        it('should check border and styles', () => {
+            checkAttributeValueTrue(standardListPg.sMultiAttr, StandardLinkData.byLineAltAttr);
+            expect(webDriver.getCSSPropertyByName(standardListPg.sMultiList, StandardLinkData.borderAttr).value)
+                .toBe(StandardLinkData.solidStyle);
+        });
+
+        it('should check content and basic interactions', () => {
+            checkElementDisplayed(standardListPg.sMultiAvatar);
+            checkElementDisplayed(standardListPg.sMultiList);
+            checkElementText(standardListPg.sMultiList);
+            checkElementDisplayed(standardListPg.sMultiToolbar);
+            checkElementText(standardListPg.sMultiToolbar);
+            checkElArrIsClickable(standardListPg.sMultiList);
+            checkElArrIsClickable(standardListPg.sMultiCheckbox);
+        });
+
+        it('should check selected item count is displayed in the toolbar', () => {
+            webDriver.scrollIntoView(standardListPg.sMultiCheckbox);
+            expect(webDriver.getText(standardListPg.sMultiToolbar)).toContain('0 : Items selected');
+            webDriver.click(standardListPg.sMultiCheckbox);
+            expect(webDriver.getAttributeByName(standardListPg.sMultiCheckbox, 'aria-checked')).toBe('true');
+            expect(webDriver.getText(standardListPg.sMultiToolbar)).toContain('1 : Items selected');
+        });
+    });
+
+    describe('Standard List Item (ByLine)- Inverted Secondary text types examples:', function() {
+        it('should check border and styles', () => {
+            checkAttributeValueTrue(standardListPg.sInvtAttr, StandardLinkData.byLineAltAttr);
+            expect(webDriver.getCSSPropertyByName(standardListPg.sInvtList, StandardLinkData.borderAttr).value)
+                .toBe(StandardLinkData.solidStyle);
+        });
+
+        it('should check content and interactions', () => {
+            checkElementDisplayed(standardListPg.sInvtAvatar);
+            checkElementDisplayed(standardListPg.sInvtList);
+            checkElArrIsClickable(standardListPg.sInvtList);
+            checkElementText(standardListPg.sInvtList);
+        });
+
+        it('should check secondary text types', () => {
+            const elCount = webDriver.getElementArrayLength(standardListPg.sInvtListItem);
+            for (let i = 0; elCount > i; i++) {
+                expect(webDriver.getAttributeByName(standardListPg.sInvtListItem, StandardLinkData.secondaryAttr, i))
+                    .not.toBe(null, '');
```

Comment:
Why do we need this check if we verify attribute value in next assertion?

Reply:
I forgot to delete one, my bad)